### PR TITLE
Remove duplicate configuration

### PR DESCRIPTION
**Pull request by zuohuadong**
Remove duplicate configuration

##### I completed the 3 steps below:

- [] I've read the [Contribution Guide](http://laradock.io/contributing).
- [] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

File: env-example [hunk 1]
```diff
@@ -78,7 +78,6 @@ PHP_FPM_INSTALL_EXIF=false
 PHP_FPM_INSTALL_AEROSPIKE=false
 PHP_FPM_INSTALL_MYSQLI=false
 PHP_FPM_INSTALL_PGSQL=false
-PHP_FPM_INSTALL_POSTGRES=false
```
Comment (appleboy) [inline env-example]: Duplicate?

Comment (zuohuadong) [reply env-example]: PHP_FPM_INSTALL_PGSQL=false
same
PHP_FPM_INSTALL_POSTGRES=false

Comment (appleboy) [reply env-example]: `PHP_FPM_INSTALL_PGSQL` for pg client.
`PHP_FPM_INSTALL_POSTGRES ` for pg server.

Comment (zuohuadong) [reply env-example]: ## see: https://github.com/laradock/laradock/blob/master/php-fpm/Dockerfile-71

54:
```
#####################################
# pgsql
#####################################

ARG INSTALL_PGSQL=false
RUN if [ ${INSTALL_PGSQL} = true ]; then \
    # Install the pgsql extension
    apt-get update -yqq && \
    docker-php-ext-install pgsql \
;fi
```


241:

```
#####################################
# postgres Modifications:
#####################################

ARG INSTALL_POSTGRES=false
RUN if [ ${INSTALL_POSTGRES} = true ]; then \
    docker-php-ext-install pgsql \
;fi
```

**Comment by Mahmoudz** [comment]
@zuohuadong thanks for the contribution
@appleboy thanks for the clarification 

I think the keys should be renamed to make it clear for others, here's just a suggestion
PHP_FPM_INSTALL_PGSQL ==> PHP_FPM_INSTALL_PGSQL_EXTENSION
PHP_FPM_INSTALL_POSTGRES  ==>  PHP_FPM_INSTALL_POSTGRES_SERVER

**Comment by zuohuadong** [comment]
see:  postgres
image

pgsql:
image

image

File: env-example (hunk 1 above)
Comment (zuohuadong) [reply env-example]:
```
#####################################
# pgsql
##################################### 
......
docker-php-ext-install pgsql 
```
for pg client?

```
#####################################
# postgres Modifications:
#####################################
......
 docker-php-ext-install pgsql 
```
 for pg server？   why？ It is  same！

**Comment by zuohuadong** [comment]
@Mahmoudz @appleboy 

Why the same code, one is the client, the other is the server?

```
#####################################
# pgsql
#####################################

ARG INSTALL_PGSQL=false
RUN if [ ${INSTALL_PGSQL} = true ]; then \
    # Install the pgsql extension
    apt-get update -yqq && \
    docker-php-ext-install pgsql \                                      <<<<<--------- see   this
;fi

```

```
#####################################
# postgres Modifications:
#####################################

ARG INSTALL_POSTGRES=false
RUN if [ ${INSTALL_POSTGRES} = true ]; then \
    docker-php-ext-install pgsql \                                     <<<<<--------- see   this
;fi
```

**Comment by Mahmoudz** [comment]
@zuohuadong I haven't checked the code, I'm referring to @appleboy comments. Let's wait his reply on this.

**Comment by zuohuadong** [comment]
@Mahmoudz 
thanks,  I want to be  developer.

## Then, I want to do:

- pgadmin : Cross-version backup .

- postgres: Cross-version upgrade.

- caddy: Like caddyserver.com as simple to install, and can be used commercially.

https://github.com/tianon/docker-postgres-upgrade

## about postgres upgrade

https://github.com/laradock/laradock/issues/1194

see： ./postgres-upgrade/Dockerfile  & docker-compose.yml

I hope you can give me some advice.

**Comment by Mahmoudz** [comment]
@zuohuadong I'll leave it to someone else to give advice regarding that. As I'm quiet busy, to dive deep in Laradock at the time. Hopefully will have free time in the next month or so. Since I have a lot of plans to enhance the project. 

I've noticed that you've been contributed a lot lately, thus I would like to list your name as one of the main contributors https://github.com/laradock/laradock#credits. Could you please provide me with your full name? Thanks,

**Comment by zuohuadong** [comment]
@Mahmoudz 
Huadong Zuo

Thanks.

**Pull request merged**